packages/vm/src/buildBlock.ts

```diff
@@ -395,7 +395,7 @@ export class BlockBuilder {
       this.checkpointed = false
     }
 
-    return block
+    return { block, requests }
```

Comment:
This is actually a great change in our breaking releases (and in this case also necessary), giving us more freedom to return contextual data when building blocks :+1: